fix: assignee bug where trap hotkeys are not getting the right prompts

codebender37 opened the pull request.

karootplx commented on commons/scoring.py:
```diff
@@ -417,6 +417,12 @@ def ground_truth_score_V1(
+        # e.g. ranks: ('cid1', 0), ('cid2', 1), ('cid3', 2), ('cid4', 3)
+        # after minmax scale: [0, 0.33, 0.667, 1]
+        # but we want the reverse, so: [1, 0.667, 0.33, 0], since cid1 is the best
+        ground_truth_arr = ground_truth_arr[::-1]
```
instead of inverting the resulting array, can we do it within the minmax_scale call. 
something like this:
`max_rank = len(cid_with_rank_sorted) - 1
ground_truth_arr = minmax_scale(
    np.array([max_rank - rank for _, rank in cid_with_rank_sorted])
).numpy()`

karootplx reviewed (approved):
lgtm

tensorplex-validator committed:
fix: trap tasks pop issue and reduce vali duels (#81)

* chore: updated logging for scoring

* fix: clamp negative weights to 0

* chore: added subtensor and watchtower update

* chore: fix the name typo in networks specification

* chore: specify local subtensor setup as optional

* chore: specifying watchtower config (#72)

* chore: specifying a startup configuraiton for watchtower

* chore: removed networks label in watchtower compose

* feat: updated scores retaining logic and weights setting intervals (#75)

* feat: updated scores retaining logic for weights setting

scores are only reset based on ScoreResetInterval and weights are
continously set based on WeightSettingInterval based on retained scores
of that period

* chore: reduced logging level for deregistered hotkeys when calculating score

* feat: used a rolling window method to calculate scores instead

* feat: updated rolling window expired votes api endpoint

* feat updated route, no longer updates task status to scored

* feat: reduce vali duels (#78)

* feat: reduce vali duels

* chore: updated comment for vali duel probability

* chore: add cli and enabl watch tower (#79)

* chore: specifying a startup configuraiton for watchtower

* chore: removed networks label in watchtower compose

* chore: specifying and enabling watchtower with scores clamping release (#73)

* chore: updated logging for scoring

* fix: clamp negative weights to 0

* chore: added subtensor and watchtower update

* chore: fix the name typo in networks specification

* chore: specify local subtensor setup as optional

* chore: specifying watchtower config (#72)

* chore: specifying a startup configuraiton for watchtower

* chore: removed networks label in watchtower compose

---------

Co-authored-by: daemontplx <daemon@tensorplex.ai>

* chore(main): release 1.2.1 (#74)

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

* chore: cli for manual weight settings

* chore: added watchtower for dev (testnet) image

---------

Co-authored-by: daemontplx <daemon@tensorplex.ai>
Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

* fix: trap tasks are scored correctly instead of being removed after first scoring (#80)

* fix: trap tasks are scored correctly instead of being removed after first scoring

the bug is: trap key will be deleted after the first scoring, and will
never be scored as trap task in subsequent scoring in the rolling-window
method

* chore: ttl will be 2x the scoring window

---------

Co-authored-by: daemontplx <daemon@tensorplex.ai>
Co-authored-by: daemontplx <165395025+daemontplx@users.noreply.github.com>
Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

github-actions[bot] committed:
chore(main): release 1.3.1 (#82)

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

daemontplx committed:
fix: assignee bug where trap hotkeys are not getting the right prompts

daemontplx committed:
fix: augmented prompt returned correctly for augmented validator duel

daemontplx committed:
Merge branch 'main' into fix/assignee-bug

daemontplx committed:
chore: reduce no vote penalty
